.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     'plugin:jest-playwright/recommended',
     'plugin:import/recommended',
   ],
-  plugins: ['prettier', 'promise', 'jest', 'anti-trojan-source'],
+  plugins: ['prettier', 'promise', 'jest', 'anti-trojan-source', 'react-native'],
```

Comment:
Move the `'react-native'` plugin down to lines 89-91, and add a `plugins: ['react-native'],` property in that override (i.e., we don't need this for every sub project).

Reply:
Made the final changes and rebased.